units/en/unitbonus1/train-play.mdx

```diff
@@ -0,0 +1,15 @@
+# Let's train and play with Huggy 🐶 [[train-play]]
+
+## Let's train Huggy 🐶
+
+Open this colab:
```

Comment:
Will we have the content of the colab in the mdx as well?

Reply:
We have a link of the colab (open in colab) in the unit. But we do not convert our colab into mdx since people need to fill the code parts. It's tutorial colabs.